src/sql/azdata.proposed.d.ts

```diff
@@ -604,6 +605,26 @@ declare module 'azdata' {
 		propertyItems?: PropertiesContainerItem[];
 	}
 
+	/**
+	 * Component to display text with an icon representing the severity
+	 */
+	export interface InfoBoxComponent extends Component, InfoBoxComponentProperties {
+	}
+
+	export type InfoBoxStyle = 'information' | 'warning' | 'error' | 'success';
+
+	/**
+	 * Properties for configuring a InfoBoxComponent
+	 */
+	export interface InfoBoxComponentProperties extends ComponentProperties {
+		style: InfoBoxStyle;
```

Comment:
I'd like us to get in the habit of documenting ALL properties/types/interfaces in this file. Not super necessary here but it doesn't hurt and having the precedent set would be nice going forward.